gtest/src/log.rs

```diff
@@ -34,6 +34,7 @@ pub struct CoreLog {
     destination: ProgramId,
     payload: Payload,
     reply_code: Option<ReplyCode>,
```

Comment:
you can just use core type ReplyDetails

Reply:
on the other hand it may decrease flexibility if I want to assert that message replied to my message and I dont care about the reply code, so could be left as is atm